README.md

```diff
@@ -100,6 +100,6 @@ You can then override the bindings for any of the following commands:
 Go to ```Preferences > Package Settings > AceJump > Settings - User```,
 and override the key ```labels```.
 
-### Hinglighting
+### Higlighting
```

Comment:
Thanks for catching this, but it looks like you missed a letter as well ;)
Update this and I'll merge.